packages/manager/src/features/Linodes/LinodeCreate/AdditionalOptions/AdditionalOptions.tsx

```diff
@@ -42,7 +45,10 @@ export const AdditionalOptions = () => {
       >
         Additional Options
       </Typography>
-      <Alerts />
+      <Stack divider={<Divider />}>
+        {flags.aclpBetaServices?.alerts && <Alerts />}
```

Comment:
```suggestion
        {showAlerts && <Alerts />}
```

issue: `showAlerts` should be used instead of `flags.aclpBetaServices?.alerts` here